src/transaction/log_tran_table.c

```diff
@@ -3543,21 +3543,15 @@ logtb_tran_update_unique_stats (THREAD_ENTRY * thread_p, const BTID * btid, int
       /* log statistics */
       char undo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
       char redo_rec_buf[3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE + MAX_ALIGNMENT];
-      RECDES undo_rec, redo_rec;
+      RECDES undo_rec;
 
       undo_rec.area_size = ((3 * OR_INT_SIZE) + OR_BTID_ALIGNED_SIZE);
       undo_rec.data = PTR_ALIGN (undo_rec_buf, MAX_ALIGNMENT);
 
-      redo_rec.area_size = ((3 * OR_INT_SIZE) + OR_BTID_ALIGNED_SIZE);
-      redo_rec.data = PTR_ALIGN (redo_rec_buf, MAX_ALIGNMENT);
-
-      btree_rv_mvcc_save_increments (btid, -n_keys, -n_oids, -n_nulls, &undo_rec);
+      btree_rv_data_pack_btid_and_stats(btid, -n_nulls, -n_oids, -n_keys, undo_rec);
```

Comment:
Is indent properly setup?
```suggestion
      btree_rv_data_pack_btid_and_stats (btid, -n_nulls, -n_oids, -n_keys, undo_rec);
```